src/pretix/control/forms/item.py

```diff
@@ -141,6 +146,39 @@ def __init__(self, *args, **kwargs):
             required=False
         )
 
+        self.fields['quota_option'] = forms.ChoiceField(
+            label=_("Quota options"),
+            widget=forms.Select,
```

Comment:
For choice fields with only three options, I feel that ``widget=forms.RadioSelect`` is better because the options are more clear to the user. What do you think? (This would also require small changes to the JavaScript part)